configs/templates/jedi-mpas-nvidia-dev/spack.yaml

```diff
@@ -1,53 +1,67 @@
-# The intent of this template is to minimize the jedi-mpas-env virtual environment
-# to provide only the packages needed to compile jedi-bundle with mpas (only).
-# Updated April 2024 by Dom Heinzeller
+# This template minimizes skylab-dev to just the required parts of the jedi-mpas-env virtual env
+# needed for compiling subsets of jedi with the compilers provided in the Nvidia HPC SDK.
+#
+# So, this template is basically a listing of the packages within jedi-mpas-env, but with some
+# packages excluded for reasons such as:
+# - python is excluded because we don't dare compile it with nvhpc, and it's just too cumbersome
+#   to list all of python's dependencies and all relevant python packages as being compiled with
+#   the fallback gcc compiler.
+#   NB: Python is needed so should be installed from the distro's package manager!
+# - gsibec is excluded because it doesn't build with nvhpc, but as it is Fortran code we don't
+#   have the option of falling back to gcc.
```

Comment:
Why not? GCC stands for GNU Compiler Collection and includes `gfortran`? Or is it that `nvhpc` complains about the `*.mod` files not being compatible between `gfortran` and `nvfortran`?

Reply:
Fortran doesn't have the concept of ABI compatibility, so you can't cross compile (unlike with C/C++ where ABI compatibility is de-facto universal even if not mandated by the standard). @l90lpa has previously verified this doesn't work.

Reply:
Ok, thanks for confirming